docs/archlinux.md

```diff
@@ -56,35 +63,39 @@ cat > packer/template.json << 'EOF'
   ],
   "builders": [
     {
+      "accelerator": "kvm",
       "type": "qemu",
       "boot_command": [
-        "<enter><wait10>",
-        "dhcpcd<enter><wait10>",
-	"usermod --password $(echo toor | openssl passwd -1 -stdin) root<enter><wait10>",
+        "<enter><wait30><enteropenssl passwd help<wait10>",
+        "dhcpcd<enter><wait5>",
+        "usermod --password $(echo root | openssl passwd -1 -stdin) root<enter><wait5>",
+        "echo 'root:magic' | chpasswd<enter><wait5>",
         "systemctl start sshd<enter>"
       ],
       "headless": true,
       "boot_wait": "10s",
       "disk_size": 1500,
       "disk_interface": "ide",
-      "http_directory": "http",
-      "iso_url": "https://mirror.archlinux32.org/archisos/archlinux-2017.04.01-i686.iso",
-      "iso_checksum": "aa4718837c95e607233aecca43824c08c798b1a4",
+      "iso_url": "file:///tmp/archlinux32-2021.12.01-i686.iso",
```

Comment:
I think it'd be nicer to leave this pointed to the url (and remove the `cp archlinux32-2021.12.01-i686.iso /tmp/` command above), so it's one less manual step.